sdk/servicefabricmanagedclusters/Microsoft.Azure.Management.ServiceFabricManagedClusters/src/Microsoft.Azure.Management.ServiceFabricManagedClusters.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.ServiceFabricManagedClusters</PackageId>
+		<Description>Microsoft Azure Management ServiceFabricManagedClusters Library</Description>
+		<AssemblyName>Microsoft.Azure.Management.ServiceFabricManagedClusters</AssemblyName>
+		<Version>0.1.0</Version>
+		<PackageTags>Microsoft Azure Management Library;ServiceFabricManagedClusters;ServiceFabricManagedClusters management;</PackageTags>
```

Comment:
Also, need to add an 'Properties\AssemblyInfo.cs' file to provide metadata for the assembly (see other projects for specifics)

Reply:
added